build.sbt

```diff
@@ -561,20 +562,23 @@ lazy val api = (project in engine("api")).
     }
   )
 
-lazy val securityApi = (project in engine("security-api")).
+lazy val security = (project in engine("security")).
   settings(commonSettings).
   settings(
-    name := "nussknacker-security-api",
+    name := "nussknacker-security",
     libraryDependencies ++= {
       Seq(
         "org.scalatest" %% "scalatest" % scalaTestV % "test",
+        "org.mindrot" % "jbcrypt" % jbcryptV,
         "com.typesafe.akka" %% "akka-http" % akkaHttpV force(),
+        "com.typesafe.akka" %% "akka-stream" % akkaV force(),
```

Comment:
Can all this stuff provided to be sure that in final security extension pack, wouldn't be those libs? Will it work with transitive dependencies?

Reply:
Good idea! Changed.